scripts/unix-install.sh

```diff
@@ -376,13 +398,16 @@ os_check()
 os_arch_check()
 {
   info "Checking for valid operating system architecture..."
-  os_arch=$(uname -m)
-  if [ "$os_arch" = 'x86_64' ]; then
-    succeeded
-  else
-    failed
-    error_exit "The operating system architecture $(fg_yellow "$os_arch") is not supported by this script."
-  fi
+  arch=$(uname -m)
+  case "$arch" in 
+    x86_64|arm64e)
```

Comment:
I think you have an extra `e` in `arm64` or am I missing something? `uname -m` return `arm64` on m1